packages/framework-integration-tests/integration/provider-specific/aws/deployment/deployment.integration.ts

```diff
@@ -0,0 +1,15 @@
+import { AWSTestHelper } from '@boostercloud/framework-provider-aws-infrastructure'
+import { applicationName } from '../../../helper/app-helper'
+import { expect } from '../../../helper/expect'
+
+describe('After deployment', () => {
+  describe('the stack', () => {
+    it('has been created successfully', async () => {
+      // The project must have been deployed by the deploy hook in setup.ts
+      // that scripts uses the cli to do the deployment, so we just check here
+      // that the Cloudformation was run by AWS successfully. For that, we can just
+      // build the AWSHelper. It will throw if the AWS stack is not ready.
+      await expect(AWSTestHelper.build(applicationName())).to.be.eventually.fulfilled
```

Comment:
This test does is provider **specific**, so we can use directly the AWSTesterHelper. 
The deploy and nuke commands are still provider-specific. Maybe we can abstract them too.